src/Mutator/MutatorResolver.php

```diff
@@ -155,8 +193,8 @@ private static function registerFromName(
     }
 
     /**
-     * @param array<string, string>|bool|stdClass $settings
-     * @param array<string, array<string, string>> $mutators
```

Comment:
Why was it changed?

Reply:
It was incorrect :)

Reply:
Hm actually lemme double check that again